manifests/l/Lenovo/QuickClean/1.02.0051/Lenovo.QuickClean.installer.yaml

```diff
@@ -1,21 +1,23 @@
-# Created with YamlCreate.ps1 v2.1.4 $debug=AUSU.7-2-6
+# Created using wingetcreate 1.1.2.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
 
 PackageIdentifier: Lenovo.QuickClean
 PackageVersion: 1.02.0051
 MinimumOSVersion: 10.0.0.0
-InstallerType: inno
 Scope: machine
 InstallModes:
 - silent
 - silentWithProgress
 InstallerSwitches:
-  Silent: /SP- /VERYSILENT /NORESTART
-  SilentWithProgress: /SP- /SILENT /NORESTART
-  InstallLocation: C:\Program Files (x86)\Lenovo\Lenovo Quick Clean
+  Silent: /VERYSILENT /NORESTART
+  SilentWithProgress: /SILENT /NORESTART
+  InstallLocation: /DIR="<INSTALLPATH>"
+  Log: /LOG="<LOGPATH>"
 Installers:
-- Architecture: x64
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Packages with x86 architecture are validated on a 32-bit OS. This seems to be a 64-bit package:
![image](https://user-images.githubusercontent.com/114952565/215599237-b3f57b77-2b0b-459d-a0da-a71f5a25b66c.png)

Reply:
Ah, ok. Wingetcreate identifies it as x86. This has happened with other packages of ours. Thanks for helping!